lib/pushmi_pullyu/swift_depositer.rb

```diff
@@ -1,13 +1,11 @@
+require 'digest/md5'
 require 'openstack'
 
 class PushmiPullyu::SwiftDepositer
 
-  attr_reader :swift_connection, :swift_container
-
-  def initialize(connection, container)
-    raise 'connection can not be nil' if connection.nil?
```

Comment:
connection can never be nil. We are always passing in a hash to this method. Even if the hash elements consist of nils like so:
```
{ username: nil, api_key: nil, etc: nil }
```
This is technically still not a nil object.